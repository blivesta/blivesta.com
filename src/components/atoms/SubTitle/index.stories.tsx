import React from 'react';
import styled from 'styled-components';
import { storiesOf } from '@storybook/react';
import SubTitle from '.';

const Wrapper = styled.div`
  width: 480px;
  padding: 60px;
`;

storiesOf(`atoms|SubTitle`, module).add(`default`, () => (
  <Wrapper>
    <SubTitle>SubTitle</SubTitle>
  </Wrapper>
));
