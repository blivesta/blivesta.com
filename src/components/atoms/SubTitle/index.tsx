import styled from 'styled-components';

const SubTitle = styled.h3`
  font-family: ${({ theme }) => theme.fonts.avantGarde};
  font-size: ${({ theme }) => theme.fontSizes[3]};
  text-transform: uppercase;
  line-height: 1;
  margin-top: 0;
  letter-spacing: 0.15em;
  text-indent: 0.15em;
`;

export default SubTitle;
