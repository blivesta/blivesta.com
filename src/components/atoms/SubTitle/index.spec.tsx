import * as React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SubTitle from '.';

describe('SubTitle', () => {
  it('snapshot', () => {
    const tree = shallow(<SubTitle>...</SubTitle>);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
