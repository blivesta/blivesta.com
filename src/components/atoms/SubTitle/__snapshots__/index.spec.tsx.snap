// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubTitle snapshot 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": false,
        "rules": Array [
          "
  font-family: ",
          [Function],
          ";
  font-size: ",
          [Function],
          ";
  text-transform: uppercase;
  line-height: 1;
  margin-top: 0;
  letter-spacing: 0.15em;
  text-indent: 0.15em;
",
        ],
      },
      "displayName": "styled.h3",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "h3",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  ...
</StyledComponent>
`;
